smac/facade/smac_bo_facade.py

```diff
@@ -20,6 +20,10 @@ class SMAC4BO(SMAC4AC):
     see smac.facade.smac_Facade for API
     This facade overwrites options available via the SMAC facade
 
+    Hyperparameters are chosen according to the best configuration for Gaussian process maximum likelihood found in
+    "Towards Assessing the Impact of Bayesian Optimization's Own Hyperparameters" by Lindauer et al., presented at the
+    DSO workshop 2019 (https://arxiv.org/abs/1908.06674).
+
```

Comment:
Could you briefly add which hyperparameters were chosen and where in that paper one would find the table?

Reply:
Unfortunately, there is no table in the paper itself, I got the table via Slack. The only thing which resembles the information is inside the ablation plots. I could reference them and add the changed hyperparameters to the docstring. Would that be sufficient? I would directly commit those to the development branch if that is okay.